docs/conf.py

```diff
@@ -57,9 +57,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = "2.2.0"
+version = "2.2.1.dev0"
```

Comment:
should this be just `.dev`?

Reply:
It should be the same. And i used bumpversion to get there...